vcnc-rest/static/Makefile.am

```diff
@@ -12,7 +12,7 @@
 all:
 	if [ ! -d $(top_srcdir)/static/swagger-ui ]; then \
 	  (cd $(top_srcdir)/static; \
-	  $(PROG_GIT) clone https://github.com/swagger-api/swagger-ui.git; \
+	  $(PROG_GIT) clone --depth 1 --branch v2.2.10 https://github.com/swagger-api/swagger-ui.git; \
```

Comment:
My point is we have no control this clone will pass successfully or not. If it failed by any reason staging would pass successfully anyway, doesn't it?